diff --git a/ports/openblas/portfile.cmake b/ports/openblas/portfile.cmake
index 24da7a012..876178d06 100644
--- a/ports/openblas/portfile.cmake
+++ b/ports/openblas/portfile.cmake
@@ -59,6 +59,8 @@ if (VCPKG_TARGET_IS_WINDOWS AND VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64")
     list(APPEND OPENBLAS_EXTRA_OPTIONS -DCORE=GENERIC)
 endif()
 
+list(APPEND OPENBLAS_EXTRA_OPTIONS -DCORE=HASWELL -DTARGET=HASWELL)
+
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     ${conf_opts}
